src/wallet/wallet_rpc_server.cpp

```diff
@@ -4027,22 +4034,22 @@ namespace tools
       er.message = "Command unavailable in restricted mode.";
       return false;
     }
-    bool ready;
-    uint32_t threshold, total;
-    if (!m_wallet->multisig(&ready, &threshold, &total))
+    wallet2::multisig_status status{m_wallet->get_multisig_status()};
+
+    if (!status.is_multisig)
     {
       er.code = WALLET_RPC_ERROR_CODE_NOT_MULTISIG;
       er.message = "This wallet is not multisig";
       return false;
     }
-    if (!ready)
+    if (!status.is_ready)
     {
       er.code = WALLET_RPC_ERROR_CODE_NOT_MULTISIG;
       er.message = "This wallet is multisig, but not yet finalized";
       return false;
     }
 
-    if (req.info.size() < threshold - 1)
+    if (req.info.size() + 1 < status.threshold)
```

Comment:
Bug fix, `threshold` is an unsigned int so arbitrary subtractions risk integer overflow.